### DISCUSSION TITLE
No matching profiles found for bundle identifier

### DISCUSSION THREAD

**Discussion by bobandrew** [codemagic-ci-cd/codemagic-docs#2280] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Hi guys, I was reading similar support topics and couldn't find a solution to fix this error in my case. I have set up my workflow in yaml according to your documentation, I followed this article https://docs.codemagic.io/partials/alternative-code-signing-methods-ios/. My goal is for Codematic to select the correct provisioning profile and self-sign the code. If there is no corresponding provisioning profile and I would like Codematic to create a new one.

My build: 658050e9aa92dc88aed71554

### Expected results

How can I fix this?

### Actual results

See steps to reproduce

### Build id (optional)

658050e9aa92dc88aed71554

**Comment by icarusdust** [comment, marked as answer]
Hi @bobandrew, the reason you are getting `No matching profiles found for bundle identifier "xxxxxxx" and distribution type "app_store"` is that you mixed up code signing settings. So, half of your YAML configuration is from manual, and the other half is from automatic signing. In short, you will need to remove the following part from `codemagic.yaml`:

```
      ios_signing:
        distribution_type: app_store
        bundle_identifier: xxxxxxxxxx
```

and continue with the doc link you shared. 

Besides, please do not skip the _Configuring environment variables_ section, otherwise you will get other errors.

**Reply by bobandrew** [reply]
@icarusdust , thansk a lot, I will test.

**Reply by bobandrew** [reply]
@icarusdust , it helped, works. Thanks a lot!
